pkg/apis/nsx.vmware.com/v1alpha1/subnet_types.go

```diff
@@ -16,7 +16,7 @@ type SubnetSpec struct {
 	// +kubebuilder:validation:Minimum:=16
 	IPv4SubnetSize int `json:"ipv4SubnetSize,omitempty"`
 	// Access mode of Subnet, accessible only from within VPC or from outside VPC.
-	// +kubebuilder:validation:Enum=Private;Public
+	// +kubebuilder:validation:Enum=Private;Public;Project
```

Comment:
Since we don't use vpcnetworkConfiguration PodAccessMode to set subnet AccessMode default value,
We can set a default value:Private here.